### PR TITLE
fix: user identity ids as string array instead of just string

### PR DESCRIPTION
**Pull request by ethanplee14**
client sessions take an array of user_identity_ids not singular

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -71,7 +71,8 @@
           }
         ],
         "@typescript-eslint/explicit-function-return-type": "off",
-        "@typescript-eslint/no-unused-vars": "off"
+        "@typescript-eslint/no-unused-vars": "off",
+        "@typescript-eslint/no-non-null-assertion": "off"
```
Comment (razor-x) [inline .eslintrc.json:75]:
```suggestion
```

This rule is intentionally enabled on this repo.

Actually, when did `no-unused-vars` get disabled>?

**Review by razor-x — CHANGES_REQUESTED** [review]

**Comment by dawnho** [comment]
Hmm, why do we need to scope a client session to multiple user identities?

File: .eslintrc.json (hunk 1 above)
Comment (razor-x) [reply .eslintrc.json:75]: You can disable specific cases if need with comments, or just add a `if (x == null) throw new Error(..)` statement

**Comment by razor-x** [comment]
> Hmm, why do we need to scope a client session to multiple user identities?

@dawnho I think this is just to match the behavior on the real API.

**Comment by ethanplee14** [comment]
> Hmm, why do we need to scope a client session to multiple user identities?

To match seam connect's behavior

File: src/lib/database/store.ts [hunk 2]
```diff
@@ -1039,7 +1043,7 @@ const initializer = immer<Database>((set, get) => ({
       ext_sdk_installation_id: params.ext_sdk_installation_id,
       phone_sdk_installation_id: installation_id,
       workspace_id: params.workspace_id,
-      user_identity_id: client_session.user_identity_id,
+      user_identity_id: client_session.user_identity_ids?.[0] ?? "",
```
Comment (razor-x) [inline src/lib/database/store.ts:1046]: Will the real api return an empty string here if there is no matching user identity?

Comment (ethanplee14) [reply src/lib/database/store.ts:1046]: No this won't return an empty string since it'll error from the check above. Will make it more explicit

Comment (ethanplee14) [reply src/lib/database/store.ts]: added explicit non null eslint ignore per line where the undefined case is thrown earlier

https://github.com/seamapi/fake-seam-connect/commit/61f50040f9c22f808c9f32f403378c5bb399bd71

**Review by razor-x — APPROVED** [review]

**Pull request merged**